code/game/objects/items/crayons.dm

```diff
@@ -110,13 +111,12 @@
 				to_chat(user, "<span class='warning'>You do not have a mouth!</span>")
 				return
 		playsound(loc, 'sound/items/eatfood.ogg', 50, 0)
-		to_chat(user, "<span class='notice'>You take a [huffable ? "huff" : "bite"] of the [name]. Delicious!</span>")
+		to_chat(user, "<span class='notice'>You take a [huffable ? "huff" : "bite"] of the [name]. Tastes of [taste_text]!")
 		user.adjust_nutrition(5)
-		if(uses)
```

Comment:
Lost feature. See line 18 of this file.

Would prob be best to make one proc for using usages. It is being done multiple times by now. Atleast here and in your newly added code.
`use` can be overriden for this. It's already a proc on item level. It takes the amount used as parameter